x-pack/docs/en/security/authentication/oidc-guide.asciidoc

```diff
@@ -552,7 +553,7 @@ for this Kibana instance.
 {kib} has in-built protection against _Cross Site Request Forgery_ attacks, which
 is designed to prevent the {kib} server from processing requests that
 originated from outside the {kib} application.
-In order to support OpenID Connect messages that originate from your
+In order to support OpenID Connect Authorization Code flow messages that originate from your
 OP or a third party (see <<third-party-login>>, we need to explicitly _whitelist_ the
```

Comment:
OP or a third party (see <<third-party-login>>, we need to explicitly _whitelist_ the
=>
OP or a third party (see <<third-party-login>>. In order to do so, we need to explicitly _whitelist_ the